.github/workflows/test.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       - name: dependencies e2e test
         working-directory: e2e/dependencies/consumer
         run: |
+          cargo t -p dependency1
```

Comment:
Need to run the dependencies' tests, luckily this only needs to be done on dependency install/update